[website] List benefits for sponsors (#39640)
mui/material-ui: docs/data/material/discover-more/backers/backers.md

<p class="description">Support the development of new open-source projects from MUI through crowdfunding.</p>

> MUI Core is our crowd-funded open-source projects, licensed under the permissive MIT license. Sponsorship increases the rate of bug fixes, documentation improvements, and feature development.

Reviewer: ```suggestion
MUI Core contains our crowd-funded open-source projects, offered under the permissive MIT license. Sponsorship increases the rate of bug fixes, documentation improvements, and feature development.
```

Author: I believe we have 3 options when it comes to the Open Collective. I went with a different approach for this page description, using option 2 below:

1. Creating one open-collective per open-source project: https://opencollective.com/mui is about Material UI, we would have a couple more to create. Considering the traction we have with the community, and the impact these donations tend to have (low), we have time until we need ones. So effectively, only rename https://opencollective.com/mui to https://opencollective.com/material-ui for now.
2. Considering that MUI is the organization, the umbrella under which we are pushing for the work. It doesn't feel so great, as a developer would likely want to sponsor one project and not another, like Vite vs. Vitest. However, it stays broad, still allowing us to cover some of Base UI, Joy UI, and MUI X.
3. Considering that MUI is one open-source project that has a large breadth. In which case, https://opencollective.com/mui wouldn't need any changes.

Even if we kill https://mui.com/joy-ui/, and we split Joy to be under Material UI (as a theme, like Material Design v3, or Material Design v2, while we work on an infra to create more theme), as well as under Base UI (as a theme), it feels like option 2 still makes the most sense right now. 

The problem with option 3 is how much it can be perceived by developers as a. lacking focus and locked-in. For example, when you look at Slack vs. Microsoft Team, you are more likely to use Slack as a standalone product, but overall Microsoft Team wins because it comes for free in a bundle you already purchased. So if we were to apply this straight to us, option 3 would make the most sense. However, our space is very different: 1. products are free 2. product value compounds with its community size. We are only left with the value of bundling. The main problem is how developers adopt the solutions that have the best execution. Headless UI failure is an example of this, Radix had x6 more execution, it was too far for Headless UI to thrive with its Tailwind CSS users.

One thing, maybe https://opencollective.com/mui -> https://opencollective.com/mui-org would make that intent to be clearer. In the same line of thought, going to GitHub https://github.com/mui/ -> https://github.com/mui-org/.

I think it's not to be confused with a change on npm @mui/material -> @material-ui/core, because there the npm org name is expected to be the open-source project name (I'm not aware of any project where it's not the case, we are the exception). On GitHub the common expectation is for it to be anything type, so we are good there.

cc @mbrookes @danilo-leal @mnajdova 

---

We are moving toward "MUI Core" to a notion of "Core of MUI", so I used more direct references to these open-source projects.

_1/3 slots available_

> Diamond Sponsors are those who have pledged \$1,500/month or more to MUI. [Tier benefits](#diamond).

Reviewer: ```suggestion
Diamond sponsors are those who've pledged \$1,500/month or more to MUI. [Tier benefits](#diamond).
```

  <a data-ga-event-category="sponsor" data-ga-event-action="docs-backers" data-ga-event-label="icons8.com" href="https://icons8.com?utm_source=MUI&utm_medium=referral&utm_content=backers" rel="noopener sponsored" target="_blank" style="margin-right: 16px;"><img height="96" width="96" src="https://images.opencollective.com/icons8/7fa1641/logo/96.png" srcset="https://images.opencollective.com/icons8/7fa1641/logo/192.png 2x" alt="Icons8" title="We provide the neat icons, photos, illustrations, and music. Developers, use our API to insert all the content we have into your apps." loading="lazy"></a>
</p>

> Gold Sponsors are those who have pledged \$500/month or more to MUI. [Tier benefits](#gold).

Reviewer: ```suggestion
Gold sponsors are those who've pledged \$500/month or more to MUI. [Tier benefits](#gold).
```

## Silver sponsors

via [Open Collective](https://opencollective.com/mui)

<p style="overflow: auto;">
  <object type="image/svg+xml" data="https://opencollective.com/mui/tiers/silver-sponsor.svg?avatarHeight=70&width=600" style="border-radius: 10px;">Silver Sponsors</object>
</p>

> Silver Sponsors are those who have pledged $250/month to $500/month to MUI. [Tier benefits](#silver).

Reviewer: ```suggestion
Silvers sponsors are those who've pledged $250/month to $500/month to MUI. [Tier benefits](#silver).
```

## Bronze sponsors

via [Open Collective](https://opencollective.com/mui)

<p style="overflow: auto;">
  <object type="image/svg+xml" data="https://opencollective.com/mui/tiers/bronze-sponsor.svg?avatarHeight=60&width=600" style="border-radius: 10px;">Bronze Sponsors</object>
</p>

> Bronze Sponsors are those who have pledged $100/month to $250/month to MUI. [Tier benefits](#sliver).

Reviewer: ```suggestion
Bronze sponsors are those who've pledged $100/month to $250/month to MUI. [Tier benefits](#sliver).
```

<span class="only-light-mode">
  <img src="/static/readme/netlify-lightmode.svg" alt="Netlify logo" loading="lazy" width="180" height="49" style="width: 100px; margin-top: 1rem;">
</span>
<span class="only-dark-mode">
  <img src="/static/readme/netlify-darkmode.svg" alt="Netlify logo" loading="lazy" width="180" height="49" style="width: 100px; margin-top: 1rem;">
</span>

[Netlify](https://www.netlify.com/) lets us distribute the documentation.

<span class="only-light-mode">
  <img src="/static/readme/browserstack-lightmode.svg" alt="BrowserStack logo" loading="lazy" width="180" height="32" style="width: 140px; margin-top: 1rem;">
</span>
<span class="only-dark-mode">
  <img src="/static/readme/browserstack-darkmode.svg" alt="BrowserStack logo" loading="lazy" width="180" height="32" style="width: 140px; margin-top: 1rem;">
</span>

[BrowserStack](https://www.browserstack.com/) lets us test in real browsers.

<img loading="lazy" alt="CodeCov logo" src="https://avatars.githubusercontent.com/u/8226205?s=70" width="70" height="70" style="width: 35px; margin-top: 1rem;">

[CodeCov](https://about.codecov.io/) lets us monitor test coverage.

## Tier benefits

### Diamond

> Your name/company logo will be placed in:

Reviewer: ```suggestion
Your company name and logo will be prominently featured:
```

Author: ```suggestion
Your organization logo will be placed in:
```

> - A prominent logo placement in sidebar of all content pages (4m+ sessions, 900k+ unique visitors per month)

Reviewer: ```suggestion
- on the sidebar of all content pages (4m+ sessions and 900k+ unique visitors per month)
```

> - ⬇️ same as the lower tiers

Reviewer: ```suggestion
- ⬇️ everywhere else as the lower tiers
```

Please contact us at sales@mui.com before subscribing to this tier to get preliminary approval.

### Gold

> Your name/company logo will be placed in:

Reviewer: ```suggestion
Your company name and logo will be featured on:
```

Author: ```suggestion
Your organization logo will be placed in:
```

- the [homepage](https://mui.com/#sponsors) (500k+ pageviews and 150k+ unique visitors per month)
- the [README.md](https://github.com/mui/material-ui#sponsors) (80k+ unique visitors per month)

> - the [sponsors' list](#gold-sponsors)

Reviewer: ```suggestion
- the [sponsor list](#gold-sponsors)
```

> \*we accept donations, but we won't show unethical ads.

Reviewer: We should probably elaborate more here—it might not be obvious what we consider "unethical."

Author: I think we can keep it broad so that we can more easily reject sponsors that don't qualify.

### Silver

> Your name/company logo will be placed in the [sponsors' list](#silver-sponsors).

Reviewer: ```suggestion
Your company name and logo will be featured on the [sponsor list](#silver-sponsors).
```

Author: This suggestion is changing the meaning, we are not showing both.

```suggestion
Your organization logo will be placed in the [sponsors' list](#sliver-sponsors).
```

### Bronze

> Your name/company logo will be placed in the [sponsors' list](#bronze-sponsors).

Reviewer: ```suggestion
Your company name and logo will be featured on the [sponsor list](#bronze-sponsors).
```

Author: ```suggestion
Your organization logo will be placed in the [sponsors' list](#bronze-sponsors).
```

### Backers

> Help support office hours and further development by becoming a Backer. We'll thank you by including your avatar in the [backers list](#backers).

Reviewer: This is the first I'm hearing about "office hours." 😅 Maybe we should add more details here about what that would be and why it would be appealing to a potential backer—sell them on it!

Author: Right, it's not clear, we can do this instead:

```suggestion
Help support more open-source development by becoming a Backer. We'll thank you by including your avatar in the [backers list](#backers).
```
